app/src/main/java/com/example/android/codeforces/Activities/ProfileActivity.java

```diff
@@ -56,4 +71,8 @@ protected void onCreate(Bundle savedInstanceState) {
             e.printStackTrace();
         }
     }
+    public void logOut(){
+        Intent intent = new Intent(this,MainActivity.class);
```

Comment:
If we give the direct intent then when we press the back button then it will come back the previous activity so, App will crashes by taking 2 usernames avoid that behaviour